common/lib/xmodule/xmodule/js/src/sequence/edit.js

```diff
@@ -0,0 +1,21 @@
+/* globals XModule, Reflect */
+
+/* Note: this was originally a one-line Coffeescript file, which we opted not to transpile directly
+ * to JS because of the grossness of the Coffeescript inheritance boilerplate. Instead I've tried to
+ * approximate the class/extends model the Coffeescript had in ES5 using reflection.
+ */
+
+(function() {
+    'use strict';
+
+    this.SequenceDescriptor = (function() {
+        function SequenceDescriptor() {
+            return Reflect.construct(XModule.Descriptor, arguments, SequenceDescriptor);
+        }
+
+        Reflect.setPrototypeOf(SequenceDescriptor.prototype, XModule.Descriptor.prototype);
```

Comment:
Why do you need to do both of these lines (feel free to just provide a link to something that explains this)?